src/audio/host.c

```diff
@@ -76,6 +76,7 @@ struct host_data {
 	uint32_t host_period_bytes;
```

Comment:
Please capture in your commit message the explanation given here:

![image](https://user-images.githubusercontent.com/62316/148053607-cea2f987-ed8b-4264-80d1-9ab395c8482d.png)

Rember that a commit message should explain WHY the commit is needed. The HOW should and WHAT should be obvious from the coded

Reply:
@dbaluta Thanks for suggestion. I have added more information in the commit message.